packages/jellyfish-transaction/src/script/dftx/dftx_balance.ts

```diff
@@ -3,17 +3,22 @@ import { BufferComposer, ComposableBuffer } from '@defichain/jellyfish-buffer'
 import { Script } from '../../tx'
 import { CScript } from '../../tx_composer'
 
-export interface TokenBalance {
+/**
+ * Known as "struct CBalances" in cpp.
+ */
+export interface TokenBalanceUInt32 {
```

Comment:
Shall we keep the old interface naming? Seems possible for downstream  transition to new implementation without any regression.

Reply:
How about we alias it, `TokenBalance = TokenBalanceUInt32`?

Reply:
Yup that works. I just don't see necessity to break downstream for this.